lib/puppet/provider/sensuctl.rb

```diff
@@ -99,7 +99,7 @@ def sensuctl_delete(*args)
     self.class.sensuctl_delete(*args)
   end
 
-  def self.sensuctl_set(command, name, property, value: nil, flags:  nil)
+  def self.sensuctl_set(command, name, property, value = nil, flags =  nil)
```

Comment:
did you confirm it fails without this?

Reply:
I did, I ran beaker acceptance tests for puppetserver without this change and got errors.